engines/sherlock/scalpel/scalpel.cpp

```diff
@@ -0,0 +1,813 @@
+	_flags[39] = true;		// Turn on Baker Street
+
+	if (!getIsDemo()) {
+	// Load the map co-ordinates for each scene and sequence data
+	_map->loadPoints(NUM_PLACES, &MAP_X[0], &MAP_Y[0], &MAP_TRANSLATE[0]);
+	_map->loadSequences(3, &MAP_SEQUENCES[0][0]);
+	}
+
+	// Load the inventory
+	loadInventory();
+
+	// Set up constants used by the talk system
+	_talk->setSequences(&TALK_SEQUENCES[0][0], &STILL_SEQUENCES[0][0], MAX_PEOPLE);
+
+	_animation->setPrologueNames(&PROLOGUE_NAMES[0], PROLOGUE_NAMES_COUNT);
+	_animation->setPrologueFrames(&PROLOGUE_FRAMES[0][0], 6, 9);
+
+	_animation->setTitleNames(&TITLE_NAMES[0], TITLE_NAMES_COUNT);
+	_animation->setTitleFrames(&TITLE_FRAMES[0][0], 7, 9);
+
+	// Starting scene
+	if (getIsDemo())
+		_scene->_goToScene = 3;
+	else
+		_scene->_goToScene = 4;
+}
+
+/**
+ * Show the opening sequence
+ */
+void ScalpelEngine::showOpening() {
+	if (getIsDemo())
+		return;
+
+	if (!showCityCutscene())
+		return;
+	if (!showAlleyCutscene())
+		return;
+	if (!showStreetCutscene())
+		return;
+	if (!showOfficeCutscene())
+		return;
+
+	_events->clearEvents();
+	_sound->stopMusic();
+}
+
+/**
+ * Show the starting city cutscene which shows the game title
+ */
+bool ScalpelEngine::showCityCutscene() {
+	byte palette[PALETTE_SIZE];
+
+	_sound->playMusic("prolog1.mus");
+	_titleOverride = "title.lib";
+	_soundOverride = "title.snd";
+	bool finished = _animation->play("26open1", 1, 255, true, 2);
+
+	if (finished) {
+		ImageFile titleImages("title2.vgs", true);
+		_screen->_backBuffer1.copyFrom(*_screen);
+		_screen->_backBuffer2.copyFrom(*_screen);
+
+		// London, England
+		_screen->_backBuffer1.transBlitFrom(titleImages[0], Common::Point(10, 11));
+		_screen->randomTransition();
+		finished = _events->delay(1000, true);
+
+		// November, 1888
+		if (finished) {
+			_screen->_backBuffer1.transBlitFrom(titleImages[1], Common::Point(101, 102));
+			_screen->randomTransition();
+			finished = _events->delay(5000, true);
+		}
+
+		// Transition out the title
+		_screen->_backBuffer1.blitFrom(_screen->_backBuffer2);
+		_screen->randomTransition();
+	}
+
+	if (finished)
+		finished = _animation->play("26open2", 1, 0, false, 2);
+
+	if (finished) {
+		ImageFile titleImages("title.vgs", true);
+		_screen->_backBuffer1.copyFrom(*_screen);
+		_screen->_backBuffer2.copyFrom(*_screen);
+
+		// The Lost Files of
+		_screen->_backBuffer1.transBlitFrom(titleImages[0], Common::Point(75, 6));
+		// Sherlock Holmes
+		_screen->_backBuffer1.transBlitFrom(titleImages[1], Common::Point(34, 21));
+		// copyright
+		_screen->_backBuffer1.transBlitFrom(titleImages[2], Common::Point(4, 190));
+
+		_screen->verticalTransition();
+		finished = _events->delay(4000, true);
+
+		if (finished) {
+			_screen->_backBuffer1.blitFrom(_screen->_backBuffer2);
+			_screen->randomTransition();
+			finished = _events->delay(2000);
+		}
+
+		if (finished) {
+			_screen->getPalette(palette);
+			_screen->fadeToBlack(2);
+		}
+
+		if (finished) {
+			// In the alley...
```

Comment:
This text should be on a black background, so in addition to fading to the screen to black it should probably clear it as well. But there I go with my premature bug reports again... :-)